content/articles/renewing-lets-encrypt-ssl-certificate.markdown

```diff
@@ -30,6 +30,7 @@ All Let's Encrypt SSL certificates, including renewals, are valid for no more th
     1.  Check the certificate [common name](/articles/what-is-common-name) matches the one you want to renew.
     1.  Check the certificate details are accurate.
     1.  Enable auto-renewal, if you want it to be auto-renewed before expiration.
+    1.  By default ECDSA is selected for you but you can select RSA as the signature algorithm if your particular scenario requires it.
```

Comment:
Same as before. We could "sell" ECDSA.

Reply:
Maybe add a link to https://blog.dnsimple.com/2022/12/ecc-support-for-letsencrypt-certificates/

Reply:
Avoid making it super long, though